### ISSUE TITLE
HTTPS request fails

### ISSUE THREAD

**Issue by radu-matei** [spinkube/containerd-shim-spin#44]
I am running an application that needs to make an outbound HTTPS request and it currently fails to send that request: `unable to get local issuer certificate `

```
Handler returned an error: Uncaught InternalError: Error::RuntimeError("POST /infer request error: error sending request for url (https://fermyon-cloud-gpu-zordxicj.fermyon.app/infer): error trying to connect: unexpected EOF (unable to get local issuer certificate)")
```

**Comment by radu-matei** [comment]
What I'm bumping into is https://github.com/deislabs/containerd-wasm-shims/issues/176.

**Comment by rajatjindal** [comment]
this is now fixed with https://github.com/spinkube/spin-operator/pull/207 and available in v0.2.0 of spin-operator.
